src/style/mono_text_style.rs

```diff
@@ -59,50 +60,87 @@ where
 {
     type Color = C;
 
-    fn render_text<D>(&self, text: &Text<'_>, target: &mut D) -> Result<(), D::Error>
+    fn render_text<D>(
+        &self,
+        text: &str,
+        mut position: Point,
+        target: &mut D,
+    ) -> Result<Point, D::Error>
     where
         D: DrawTarget<Color = Self::Color>,
     {
-        target.draw_iter(MonoPixels::new(text, *self))
+        let mut first = true;
+
+        for c in text.chars() {
+            if first {
+                first = false;
+            } else if F::CHARACTER_SPACING > 0 {
+                // File space between characters if background color is set.
```

Comment:
```suggestion
                // Fill space between characters if background color is set.
```

Reply:
:grimacing: Another stupid typo, I shouldn't write comments at the moment.